.gitlab/kernel_version_testing/security_agent.yml

```diff
@@ -0,0 +1,266 @@
+# System probe overrides
+upload_dependencies_security_agent_x64:
+  extends:
+    - .package_dependencies
+  needs: ["pull_test_dockers_x64"]
```

Comment:
CWS tests do not depend on docker images like that AFAIK, only USM tests

Reply:
Maybe we can have a different [vmconfig](https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/system-probe/config/vmconfig.json) template file for CWS to prevent the docker-disk to be downloaded in their tests.